src/WebJobs.Extensions.DurableTask/EventSourceListener.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics.Tracing;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class EventSourceListener : EventListener
+    {
+        private readonly LinuxAppServiceILogger logger;
+
+        public EventSourceListener(LinuxAppServiceILogger logger)
+        {
+            this.logger = logger;
+        }
+
+        protected override void OnEventSourceCreated(EventSource eventSource)
+        {
+            if (eventSource.Name == "DurableTask-Core" ||
+                eventSource.Name == "DurableTask-AzureStorage" ||
+                eventSource.Name == "WebJobs-Extensions-DurableTask")
+            {
+                this.EnableEvents(eventSource, EventLevel.LogAlways, EventKeywords.All);
```

Comment:
@cgillum, I know you cleaned up the noise in the DurableTask-Core logs, but are we also looking for verbose logs here too?

Reply:
Heh, so there are actually two event sources named `DurableTask-Core` now 😅. We definitely want to only write events emitted from the newer one, which is the "cleaned up" one. We can filter out the "old" one by checking `eventSource.Guid != new Guid("7DA4779A-152E-44A2-A6F2-F80D991A5BEE")`.

Reply:
Regarding Connor's point about verbosity - that's a good question. We log everything on Windows but that's much lower overhead compared to Linux. It makes me wonder if we should add some environment variable switches that allow us to dial down some of this logging. I'm not sure what the right way to approach this is, though, so perhaps we should consider it for a separate PR.